install_script.sh.template

```diff
@@ -268,11 +268,12 @@ function ensure_config_file_exists() {
     local sudo_cmd="$1"
     local config_file="$2"
     local owner="$3"
+    local valid_condition="$4"
 
     if [ -e "$config_file" ]; then
         printf "\033[34m\n* Keeping old $config_file configuration file\n\033[0m\n"
         return 1
-    else
+    elif [ -z "$valid_condition" ] || [ "$valid_condition" == "true" ]; then
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏
If `DD_RUNTIME_SECURITY_CONFIG_ENABLED` is not defined should we create the config file?

Reply:
you are right, I changed the condition, it should be better now